lib/services/app-files-updater.ts

```diff
@@ -17,7 +17,7 @@ export class AppFilesUpdater {
 
 		// exclude the app_resources directory from being enumerated
 		// for copying if it is present in the application sources dir
-		const appResourcesPathNormalized = path.normalize(projectData.appResourcesDirectoryPath + "\\");
+		const appResourcesPathNormalized = path.normalize(projectData.appResourcesDirectoryPath + path.sep);
```

Comment:
why do you need `/` at the end ?

Reply:
So that there are no false-positives when there is a directory which contains similar name.

For example, if your resources remain as `App_Resources`, but you've got another `App_Resources123` directory, you don't want that filtered out. Similarly if you rename the resources directory to something more generic like `resources`, and if you have another `resources_home_page`, the latter should not be filtered out of the application files.

Reply:
@Pip3r4o added it so we can filter more accurately. For example if you have Resources folder named "native" and folder named "native_classes" we want to filter only files inside "native".